modules/ycc_vm.py

```diff
@@ -566,11 +567,24 @@ def _translate(self):
                         disk["size"] = disk["size"] * 2 ** 30
 
         if params.get("image_family"):
-            params["image_id"] = self.image_service.GetLatestByFamily(
-                GetImageLatestByFamilyRequest(
-                    folder_id="standard-images", family=params.get("image_family")
-                )
-            ).id
+            try:
+                params["image_id"] = self.image_service.GetLatestByFamily(
+                    GetImageLatestByFamilyRequest(
+                        folder_id="standard-images", family=params.get("image_family")
+                    )
+                ).id
+            except _InactiveRpcError as err:
+                if (
+                        err._state.code  # pylint: disable=protected-access
+                        is StatusCode.NOT_FOUND
+                ):
+                    params["image_id"] = self.image_service.GetLatestByFamily(
+                        GetImageLatestByFamilyRequest(
+                            folder_id=params.get("folder_id"), family=params.get("image_family")
+                        )
+                    ).id
+                else:
+                    raise err
```

Comment:
I think is not really needed to use `params.get` because u already have an `if param,get` before whole block.